### PR TITLE
boards: m5stack: cores3: Add RTC configuration

### PR DESCRIPTION
**Pull request by soburi**
Adding PFC8563 configuration for enabling RTC feature.

### CODE REVIEW

**Comment by soburi** [comment]
@sylvioalves  @marekmatej 

I know you're busy, but I couldn't think of anyone else who would be suitable to review this, so I'd like you to review this as well.

**Comment by sylvioalves** [comment]
> @sylvioalves @marekmatej
> 
> I know you're busy, but I couldn't think of anyone else who would be suitable to review this, so I'd like you to review this as well.

Hey, don't bother, ping us whenever you need. :)

File: boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_common.dtsi [hunk 1]
```diff
@@ -68,6 +70,12 @@
 	clock-frequency = <I2C_BITRATE_STANDARD>;
 	pinctrl-0 = <&i2c0_default>;
 	pinctrl-names = "default";
+
+	pfc8563_rtc: pfc8563@51 {
```
Comment (sylvioalves) [inline boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_common.dtsi:74]: I think this M5Stack board uses `bm8563` PN. If so, would you mind renaming `pfc8563` to `bm8563` instead? 

```c
	bm8563: bm8563@51 {
		compatible = "nxp,pcf8563";
		reg = <0x51>;
		status = "okay";
	};
```

Comment (soburi) [reply boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_common.dtsi]: Oops, you're correct.
I referred to core2, but I'll fix it as follow the documentation.

Thank you for your review.

**Review by sylvioalves — APPROVED** [review]

**Review by kartben — APPROVED** [review]

**Review by marekmatej — APPROVED** [review]

**Pull request merged**